### DISCUSSION TITLE
use PAT in docs - GITHUB_TOKEN

### DISCUSSION THREAD

**Discussion by Portal3d** [community/community#44252] [Enterprise] [answered]
### Select Topic Area

Question

### Body

When a Github Actions workflow is started, the docs state that the Github Actions app sets a new short-lived token via the `GITHUB_TOKEN` at every workflow run. The permissions linked to this token are controlled by [repository](https://docs.github.com/en/repositories/managing-your-repositorys-settings-and-features/enabling-features-for-your-repository/managing-github-actions-settings-for-a-repository#setting-the-permissions-of-the-github_token-for-your-repository)/[organization](https://docs.github.com/en/organizations/managing-organization-settings/disabling-or-limiting-github-actions-for-your-organization#setting-the-permissions-of-the-github_token-for-your-organization)/[enterprise](https://docs.github.com/en/enterprise-server@3.5/admin/policies/enforcing-policies-for-your-enterprise/enforcing-policies-for-github-actions-in-your-enterprise#enforcing-a-policy-for-workflow-permissions-in-your-enterprise) settings, but can be [tweaked via the `permissions` settings](https://github.blog/changelog/2021-04-20-github-actions-control-permissions-for-github_token/) since early 2021.

Isn't it ill adviced then to still leverage a (long lived) Personal Access Token if you want more permissions? Are there still cases where this would be needed?
 
This is what I still find in the docs: https://docs.github.com/en/actions/security-guides/automatic-token-authentication#granting-additional-permissions

**Comment by GabrielVitorGL** [comment, marked as answer]
* Accessing a private repository other than the one running the workflow. The `GITHUB_TOKEN` is scoped to the repository, so it can't do that.

* Making events caused by the workflow (e.g. a push to the repository) trigger additional workflows. Activities authorized with the `GITHUB_TOKEN` never do that to avoid unintentional recursive runs.
